### ISSUE TITLE
Tab focus breaks after choosing a suggested artist

### ISSUE THREAD

**Issue by kafkaesc** [kafkaesc/Rhythm-Run#97]
For keyboard-centric users, when they tab over one of the artists in the Suggested Artists sections and click enter, the selected artist is added to the list of selected artists correctly.

However, the tab focus returns all the way to the first tab index in the page--usually the "Rhythm Run" heading or the menu icon for the Drawer. This breaks the keyboard navigation of the page and is a violation of WCAG AA, which this project aims to maintain.

When a user selects a suggested artists, the keyboard focus should shift to the previous artist in the list, or the previous tabbable item in the page if Suggested Artists is only one artist.
